os/drivers/compression/compress.c

```diff
@@ -158,6 +158,7 @@ static int comp_ioctl(FAR struct file *filep, int cmd, unsigned long arg)
 		if (ret == ENOMEM) {
 			bcmpdbg("Output buffer allocated is not sufficient\n");
 		}
+		break;
```

Comment:
it's not related to this pr but please check code line where return just ERROR, it is wrong. each of ioctl should return one of errno.